src/resources/js/components/ListPresentations.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
```

Comment:
not vuetify compliant `<div class="container">` should be replaced by `<v-container>` 
has the class container is a bootstrap css style and we are not using it

Reply:
ok, I changed it in [fbe572](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/fbe5728e1814e7da7502a2a1fb5702f6ab5d2281)